src/metabase/api/dataset.clj

```diff
@@ -157,10 +159,18 @@
 #_{:clj-kondo/ignore [:deprecated-var]}
 (api/defendpoint-schema POST "/native"
   "Fetch a native version of an MBQL query."
-  [:as {query :body}]
+  [:as {{:keys [database pretty?] :as query} :body}]
+  {database s/Int
```

Comment:
nbd, but we normally use `ms/PositiveInt`

Reply:
er, this is plumatic, not malil, so `su/IntGreaterThanZero`. But it'd be nice to convert it to a malli defendpoint

```
{database ms/IntGreaterThanZero
 pretty?  [:maybe :boolean]}
 ```

Reply:
Good idea.